packages/block-library/src/heading/edit.js

```diff
@@ -54,17 +54,19 @@ function HeadingEdit( {
 					onChange={ ( newLevel ) =>
 						setAttributes( { level: newLevel } )
 					}
+					isCollapsed={ Platform.OS === 'web' }
```

Comment:
Do mobile toolbars support isCollapsed prop? Wht happens if we keep it true for mobile too?

Reply:
They don't show., it looks collapsed is equivalent to hidden in mobile.

Reply:
Do you think we should update the toolbar implementation to just ignore that prop instead?

Reply:
I can update the code for the native ToolbarGroupCollapsed to simple do the same as the non-collapsed version, until we get a proper implementation of a collapsed toolbar for mobile.